### PR TITLE
dev-ruby/gh: initial commit

### PR DESCRIPTION
**Pull request by junghans**
Last missing dependency for https://github.com/travis-ci/travis.rb, @gentoo/ruby please review as ruby ebuilds are usually not my cup of tea.

### CODE REVIEW

**Comment by monsieurp** [comment]
@junghans we know ruby ebuilds are not your cup of tea, no need to reiterate that fact in each PR. ;)

**Comment by monsieurp** [comment]
@graaff please.

**Comment by graaff** [comment]
By now that cup of tea should be changing anyway :-)

File: dev-ruby/gh/gh-0.14.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+ruby_add_bdepend "
+	dev-ruby/rspec
+	dev-ruby/webmock
+"
+
+ruby_add_rdepend "
```
Comment (graaff) [inline dev-ruby/gh/gh-0.14.0.ebuild]: Some of these dependencies are versioned, that also needs to be mimicked here. rubygems will check the gemspec versions and won't load if things don't match up. See e.g. https://rubygems.org/gems/gh

File: dev-ruby/gh/gh-0.14.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+USE_RUBY="ruby20 ruby21"
+RUBY_FAKEGEM_RECIPE_DOC="rdoc"
+RUBY_FAKEGEM_RECIPE_TEST="none"
```
Comment (graaff) [inline dev-ruby/gh/gh-0.14.0.ebuild]: There are specs and you add dependencies for them, but they are not being run? You probably want the rspec3 recipe here (and then the explicit rspec bdepend can be dropped).

**Comment by junghans** [comment]
Sorry, I am always using the same template ;-) Will start talking about cups of coffee soon!

File: dev-ruby/gh/gh-0.14.0.ebuild (hunk 1 above)
Comment (junghans) [inline dev-ruby/gh/gh-0.14.0.ebuild]: I saw that, but we didn't have the older versions in the tree.

File: dev-ruby/gh/gh-0.14.0.ebuild (hunk 2 above)
Comment (junghans) [inline dev-ruby/gh/gh-0.14.0.ebuild]: Fixed!

File: dev-ruby/gh/gh-0.14.0.ebuild (hunk 1 above)
Comment (junghans) [inline dev-ruby/gh/gh-0.14.0.ebuild]: Fixed!

**Comment by graaff** [comment]
:+1:

**Pull request merged**